docs/dasharo-tools-suite/overview.md

```diff
@@ -24,7 +24,35 @@ Thank you for using Dasharo Tools Suite. If you have encountered any problems
 with this system or would like to provide feedback for us - please open an issue
 on [Dasharo
 issues](https://github.com/Dasharo/dasharo-issues/issues?q=is%3Aopen+is%3Aissue+label%3ADasharoToolsSuite).
+Dasharo Tools Suite has its own label: `DasharoToolsSuite` and issue template
+for bugs: `DTS bug report`. Other types of issues in the repository are generic
+and can be used with the `DasharoToolsSuite` label.
 
 And if you have already used this system and would be interested in supporting
 the project, please check the
 [How to support us?](../osf-trivia-list/dts.md#how-to-support-us) section.
+
+## Gathering additional information and providing logs
```

Comment:
I would omit this section. If really required, I would move it to bug report.

Reply:
Amended here b07c88d5d094dcab8b4de9bbec664d15942e09fc.